cmd/use/namespace.go

```diff
@@ -0,0 +1,81 @@
+package use
+
+import (
+	"github.com/devspace-cloud/devspace/pkg/devspace/config/configutil"
+	"github.com/devspace-cloud/devspace/pkg/devspace/config/generated"
+	"github.com/devspace-cloud/devspace/pkg/util/kubeconfig"
+	"github.com/devspace-cloud/devspace/pkg/util/log"
+	"github.com/mgutz/ansi"
+	"github.com/spf13/cobra"
+)
+
+type namespaceCmd struct{}
+
+func newNamespaceCmd() *cobra.Command {
+	cmd := &namespaceCmd{}
+
+	useNamespace := &cobra.Command{
+		Use:   "namespace",
+		Short: "Tells DevSpace which namespace to deploy to",
```

Comment:
I'm not sure if we need this command since in my eyes it makes things more complex for the user and us. The main problem with this command (and `devspace use space`) is that it hides what kube context and namespace is used during deployment. While for `devspace use space` this is partly encouraged since the kube context and namespace are generated by us, a command like this would lead to much confusion since a user doesn't sees the used kube context or namespace in his `devspace.yaml` anymore and has to guess which namespace and kube context he had configured.  

If we decide for such a command, I would prefer to change the devspace.yaml directly like other configure commands are doing which is much more transparent for the user and also makes the code a lot simpler.